tools/adventure-pack/goodies/typescript/Iterator.prototype.some/index.test.ts

```diff
@@ -1,5 +1,8 @@
 import { describe, expect, it, jest } from "@jest/globals";
 
+import { iteratorPrototype } from "../Iterator.prototype";
+delete (iteratorPrototype as unknown as Record<string, unknown>).some;
```

Comment:
What about `.every`?

Reply:
I didn't realize we already had an `every`